spec/helpers/tests2png/painter.js

```diff
@@ -1,3 +1,4 @@
+/*eslint-disable no-param-reassign, no-use-before-define*/
```

Comment:
`no-use-before-define` is disabled due to recursive calling between `drawNestedObservable` and `drawObservableMessages`

`no-param-reassign` is disable due to reassign to `out` parameters.

Maybe both of them can be refactored instead?